test/oph/ehoks/hoks/hoks_save_test.clj

```diff
@@ -459,21 +459,6 @@
             (ah/get-tarkentavat-tiedot-osaamisen-arvioija (:id tta)))
           (assoc data :aiemmin-hankitun-osaamisen-arvioijat [])))))
 
-(deftest get-hoks-test-send-msg-fail
-  (testing
-   "Save HOKS but fail in sending msg, test that HOKS saving is rolled back"
-    (with-redefs [oph.ehoks.external.aws-sqs/send-amis-palaute-message
-                  #(throw (Exception. "fail"))]
-      (is (thrown? Exception (h/save-hoks! hoks-data)))
```

Comment:
Olisko tää testi hyvä säilyttää, jos vaan poistaa tämän rivin? Vaihtaa vaan testin kuvausta -> "Save HOKS but fail in sending msg, test that HOKS saving is not rolled back"

Reply:
Miltäs nyt näyttää?